src/store/reducers.js

```diff
@@ -124,49 +125,50 @@ const createRootReducer = routeHistory =>
 
     application: combineReducers({
       ui: combineReducers({
-        editPlaylistModal,
+        appCTAModal,
```

Comment:
nice! thx. FYI we have a rough plan to reorganize/further break down some of these combinations:
https://www.notion.so/audiusproject/Frontend-Store-Architecture-5e6254c33b2b4be0ae46b6396732b1ef